Frontend/library/src/PeerConnectionController/AggregatedStats.ts

```diff
@@ -33,6 +33,7 @@ export class AggregatedStats {
     sessionStats: SessionStats;
     streamStats: StreamStats;
     codecs: Map<string, string>;
+    transport: RTCTransportStats;
```

Comment:
Consider renaming the field `transportStats` to be consistent with the other fields here.